catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -161,23 +175,24 @@ public Metacard getMetacard(String id)
         return getMetacards(Metacard.ID, ids, tagFilter);
     }
 
-    public Map<String, Result> getMetacards(Collection<String> ids, Filter tagFilter)
+    public Map<String, Result> getMetacards(List<String> writableSources, Collection<String> ids, Filter tagFilter)
             throws UnsupportedQueryException, SourceUnavailableException, FederationException {
-        return getMetacards(Metacard.ID, ids, tagFilter);
+        return getMetacards(writableSources, Metacard.ID, ids, tagFilter);
     }
 
     public Map<String, Result> getMetacards(String attributeName, Collection<String> ids,
             String tag)
             throws UnsupportedQueryException, SourceUnavailableException, FederationException {
-        return getMetacards(attributeName,
+        return getMetacards(null,
```

Comment:
In my opinion, I don't see any benefit in creating an empty reference when passing null suffices